.github/workflows/cancel_redundant_workflows.yml

```diff
@@ -0,0 +1,20 @@
+name: Cancel redundant workflows
+on:
+  workflow_run:
+    types:
+      - requested
+    workflows:
```

Comment:
This is the most lightweight way for us to cancel redundant workflows, but it comes at the potential cost of needing to list out all the names of our workflows here. It would not be too difficult to write a script that would generate the names, but I wonder if there's a better, more GH supported way to accomplish the same thing. (e.g., if there were some way to just specify all workflows with a *).